frontend/src/components/SelfEvaluation/index.js

```diff
@@ -1,25 +1,53 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { Form } from "@code4ro/taskforce-fe-components";
-import data from "../../data/personal-assesment.json";
-import { submitFormResults } from "../../api/evaluation";
+import Evaluation from "../Evaluation";
+import IntroSelfEvaluation from "./introSelfEvaluation";
+import EvaluationApi from "../../api/evaluationApi";
+import FinishFormButton from "../Evaluation/finishFormButton";
 
 const SelfEvaluation = () => {
```

Comment:
This component and otherEvaluation are very similar - can we merge them somehow?

Reply:
I was wondering about. Stopped short of it - will give it a try to see how readable it it